tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -178,11 +178,6 @@ def clock_yield(generator):
     test_flag = True
 
 
-@cocotb.test(expect_fail=True)
-def test_duplicate_yield(dut):
-    """A trigger can not be yielded on twice"""
```

Comment:
Note for myself: would be good to check when this was introduced in order to guess what was intended to be tested here.

Reply:
Looks like it was just commited unfinished 13128df6bcb6062b12b660ad67b50b2e36df8680.

Reply:
I think it means yielding on a list of triggers where the triggers are not unique.

Reply:
Ah, that could be it. Would be interesting to see what happens in that case, but obviously out of scope here.